src/sentry/workflow_engine/service/action/impl.py

```diff
@@ -26,3 +27,31 @@ def update_action_status_for_organization_integration(
             integration_id=integration_id,
             dataconditiongroupaction__condition_group__organization_id=organization_id,
         ).update(status=status)
+
+    def update_action_status_for_sentry_app(
+        self,
+        *,
+        organization_id: int,
+        status: int,
+        sentry_app_install_uuid: str | None = None,
+        sentry_app_id: int | None = None,
```

Comment:
I'd prefer we break this into 2 separate RPC methods if these are mutually exclusive params. Core logic could still live in a helper method though to keep most of the logic simple.